app/_components/commissionHistory/CommissionHistory.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+interface CommissionItem {
+  id: number;
+  DeveloperName: string;
+  Price: string;
+  code: string;
+  rate:number;
+}
+
+interface CommissionHistoryProps {
+  data: CommissionItem[];
+}
+
+const CommissionHistory: React.FC<CommissionHistoryProps> = ({ data }) => {
```

Comment:
Are you sure that's right ?
you're mapping on data inside and outside the component